### PR TITLE
Fix push events

### PR DESCRIPTION
**Pull request by mythsunwind**

### CODE REVIEW

File: src/main/java/com/wire/bots/github/model/Response.java [hunk 1]
```diff
@@ -33,4 +33,7 @@
 
     @JsonProperty("repository")
     public Repository repository;
+
+    @JsonProperty("created")
+    public Boolean created;
```
Comment (mythsunwind) [inline src/main/java/com/wire/bots/github/model/Response.java:38]: Oh the Boolean!

**Review by mythsunwind — COMMENTED** [review]
Oh ha!

File: src/main/java/com/wire/bots/github/resource/GitHubResource.java [hunk 2]
```diff
@@ -109,31 +110,28 @@ private void handleIssue(String event, WireClient client, Response response) thr
...
 
     private void handlePrReview(WireClient client, Response response) throws Exception {
-        if(response.action == null) return;
+        if (response.action == null) return;
 
         switch (response.action) {
             case "submitted": {
                 String title = "";
-                if("".equals(response.review.body)) {
+                if ("".equals(response.review.body)) {
```
Comment (dkovacevic) [inline src/main/java/com/wire/bots/github/resource/GitHubResource.java]: why not: response.review.body.isEmpty()?

Comment (mythsunwind) [reply src/main/java/com/wire/bots/github/resource/GitHubResource.java]: Because if it is null we have a nullpointerException

Comment (dkovacevic) [reply src/main/java/com/wire/bots/github/resource/GitHubResource.java]: I see... damn

**Pull request merged**